.github/workflows/runtime-dockerhub.yml

```diff
@@ -0,0 +1,29 @@
+name: Build docker image and deploy
+
+on:
+  push:
+    branches: main
+    paths:
+      - environments/**
+  pull_request: # only to test while developing
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        image: [runtime_0-0-3_py39]
```

Comment:
Currently this is pushing to `coiled/runtime_0-0-3_py39:latest` on docker hub. However, I think `coiled/coiled-runtime-py39:0.0.3` uses a more conventional name. @ncclementi thoughts on using this naming convention instead?

Reply:
yes, using 0.0.3 seems a better choice. Now that things are working I will get to move things to the software yaml and use build-args. Thank you @jrbourbeau for troubleshooting the credential issue.

Reply:
After looking at [Dask's docker images](https://hub.docker.com/r/daskdev/dask/tags), I'd like to revise my previous suggestion and propose we go with `coiled/coiled-runtime:0.0.3-py3.9` instead. This way, we only have a single docker repository for all our runtime images and all the additional information on needs (i.e. runtime + Python version) are specified in the tag. What do you think @ncclementi?

Reply:
That seems reasonable to me. Happy to modify the code accordingly.

Reply:
I think we will need to give the bot access to the right repo in dockerhub, @jrbourbeau  can you take care of that before I push to see if it works?

Reply:
Yep, on it now. I just created a new `coiled/coiled-runtime` repo in the `coiled` docker hub org (see https://hub.docker.com/repository/docker/coiled/coiled-runtime). The bot should have permissions to push to that repo too